providers/prefab/README.md

```diff
@@ -0,0 +1,56 @@
+# Unofficial Prefab OpenFeature Provider for GO
+
+[Prefab](https://www.prefab.cloud/) OpenFeature Provider can provide usage for Prefab via OpenFeature GO SDK.
+
+## Installation
+
+To use the provider, you'll need to install [Prefab Go client](https://github.com/prefab-cloud/prefab-cloud-go) and Prefab provider. You can install the packages using the following command
+
+```shell
+go get github.com/prefab-cloud/prefab-cloud-go
+go get github.com/open-feature/go-sdk-contrib/providers/prefab
+```
+
+## Usage
+Prefab OpenFeature Provider is using Prefab GO SDK.
+
+### Usage Example
+
+```go
+import (
+  provider "github.com/open-feature/go-sdk-contrib/providers/prefab/pkg"
+  of "github.com/open-feature/go-sdk/openfeature"
+  prefab "github.com/prefab-cloud/prefab-cloud-go/pkg"
+)
+
+of.SetProvider(provider)
```

Comment:
I'm not sure if this is a bug or if the README omits the full code intentionally, but I found it a little confusing that I couldn't just copy directly from the README and have it work. 

Here's what did work for me:

```go
	config := provider.ProviderConfig{
		APIKey: "YOUR_API_KEY_GOES_HERE",
	}

	prefab, err := provider.NewProvider(config)
	if err != nil {
		fmt.Println(err)
		return
	}

	of.SetProvider(prefab)
```

Reply:
I updated the readme, it should now work as is.